### ISSUE TITLE
Update expected output

### ISSUE THREAD

**Issue by MLopezJ** [MLopezJ/asset-tracker-cloud-coiote-azure-converter-js#9]
# TL;DR
Issue to explain why { 'pmin': 600, 'pmax': 3600 } should not be present on the [Expected Output](https://github.com/MLopezJ/asset-tracker-cloud-coiote-azure-converter-js#expected-output). 

Change addressed here: https://github.com/MLopezJ/asset-tracker-cloud-coiote-azure-converter-js/commit/128038853cfbe9bc45ce676d776de64733528a5a

# Issue discussion

Expected output is following a no desired format in the next objects 

| object URN | prop | actual value | expected |
| --- | --- | --- | --- | 
| 6 | 0 | { 'pmin': 600, 'pmax': 3600 }| undefined |
| 6 | 1 | { 'pmin': 600, 'pmax': 3600 }| undefined |
| 3303:1.1 | 5700 | { 'pmin': 600, 'pmax': 3600 }| undefined |
| 3304:1.1 | 5700 | { 'pmin': 600, 'pmax': 3600 }| undefined |

`* At this point, the expected value is undefined but it could change in the future `

## More info 

     { 'pmin': 600, 'pmax': 3600 }
     
     Is this struct defined on `LwM2M` ?

No. All expected values from those props should be **float** type

> * `Object` 6 `Prop` 0
> * `Object` 6 `Prop` 1
> * `Object` 3304:1.1 `Prop` 5700
> * `Object` 3315 `Prop` 5700

| ID  |  Object  | Prop | Name | Expects | LwM2M Def |
|---|---|---|---|---|---|
|  6 |  Location | 0 | Latitude | **float** |  [link](https://github.com/OpenMobileAlliance/lwm2m-registry/blob/prod/6.xml) | 
|  6 |  Location | 1 | Longitude | **float** |  [link](https://github.com/OpenMobileAlliance/lwm2m-registry/blob/prod/6.xml) | 
| 3304  | Humidity  | 5700  | Sensor Value | **float** |  [link](https://github.com/OpenMobileAlliance/lwm2m-registry/blob/prod/3304.xml)  | 
| 3315  | Barometer | 5700 | Sensor Value | **float** |   [link](https://github.com/OpenMobileAlliance/lwm2m-registry/blob/prod/3315.xml) |

_Originally posted by @MLopezJ in https://github.com/MLopezJ/asset-tracker-cloud-coiote-azure-converter-js/issues/7#issuecomment-1424426763_
